policyengine_us/parameters/gov/states/mn/tax/income/amt/fractional_income_threshold.yaml

```diff
@@ -5,24 +5,40 @@ metadata:
   unit: currency-USD
   breakdown:
   - filing_status
+  uprating:
```

Comment:
we need to add the uprating to each individual filing status - for all breakdown parameters